tests/realm.cpp

```diff
@@ -1109,7 +1103,7 @@ TEST_CASE("SharedRealm: SchemaChangedFunction") {
             };
             r1->update_schema(new_schema, 2);
             REQUIRE(schema_changed_called == 1);
-            REQUIRE(changed_fixed_schema.find("object3")->property_for_name("value")->table_column == 0);
+            // REQUIRE(changed_fixed_schema.find("object3")->property_for_name("value")->table_column == 0);
```

Comment:
These should probably all just check that `column_key` is set to an actual value (without caring what that value is).